tgui/packages/tgui/interfaces/Vending.js

```diff
@@ -134,7 +131,7 @@ export const Vending = (props, context) => {
                 key={product.name}
                 custom={custom}
                 product={product}
-                productStock={data.stock[product.path]} />
+                productStock={data.stock[product.name].amount} />
```

Comment:
I *explicitly* changed this to use path and not name.
https://github.com/BeeStation/BeeStation-Hornet/pull/7229

Do not undo this

Reply:
fixed